schema/core.md

```diff
@@ -673,63 +617,66 @@ are used for a particular study.
 Note: `hasPart` and `isPartOf` are generally used for records that are grouped
 together or records that belong to other groupings. For example, `hasPart` may
 be used to indicate a record that consists of one or more additional
-CreativeWork (this is similar to a HydroShare collection). In contrast,
+`CreativeWork` (this is similar to a HydroShare collection). In contrast,
 `isPartOf` is used to indicate catalog records that reference a designated
-CreativeWork. As such, these are not demonstrated in the example below.
+`CreativeWork`. 
 
+The example below shows a simple schema for a `CreativeWork` that links to another
+`CreativeWork`.
```

Comment:
Rephrase this sentence to something like:

> The following example illustrates how a `CreativeWork` may reference another `CreativeWork` in the catalog. This can be read as, the "Collection of Sample Datasets for IGUIDE" is a collection of record(s) that includes the "IGUIDE Shapefile Testing Resource".

Reply:
commit eb0655c